src/server.c

```diff
@@ -4031,7 +4024,7 @@ int processCommand(client *c) {
 
     /* in case we are starting to ProcessCommand and we already have a command we assume
      * this is a reprocessing of this command, so we do not want to perform some of the actions again. */
-    int client_reprocessing_command = c->cmd ? 1 : 0;
+    int client_reprocessing_command = c->flag.reprocessing_command ? 1 : 0;
```

Comment:
Do we still need this variable? can't we just check the flag?

Reply:
I just aim at reducing the code change here